components/script/dom/htmlformelement.rs

```diff
@@ -302,17 +300,18 @@ impl HTMLFormElement {
 
             // Step 5
             // https://tools.ietf.org/html/rfc7578#section-4
-            result.push_str(&*format!("\r\n--{}\r\n", boundary));
+            let mut boundary_bytes = format!("--{}\r\n", boundary).into_bytes();
```

Comment:
Could you leave a comment here and file a spec bug? I also suspect that it's very unlikely that this spec is wrong here since there are many implementors -- what do other servers do with this?